File: ios/Sources/ScanditIdAamvaBarcodeVerificationNativePlugin/ScanditIdAamvaBarcodeVerificationNativePlugin.swift
import Capacitor
import Foundation

/// Please read the Capacitor iOS Plugin Development Guide
/// here: https://capacitorjs.com/docs/plugins/ios
@objc(ScanditIdAamvaBarcodeVerificationNativePlugin)
public class ScanditIdAamvaBarcodeVerificationNativePlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "ScanditIdAamvaBarcodeVerificationNativePlugin"
    public let jsName = "ScanditIdAamvaBarcodeVerificationNative"
    public let pluginMethods: [CAPPluginMethod] = [
        CAPPluginMethod(name: "echo", returnType: CAPPluginReturnPromise)
    ]
    @objc func echo(_ call: CAPPluginCall) {
        call.resolve()
    }
}

File: android/src/main/java/com/scandit/capacitor/datacapture/id_aamva_barcode_verification/ScanditIdAamvaBarcodeVerificationNativePlugin.java
package com.scandit.capacitor.datacapture.id_aamva_barcode_verification;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "ScanditIdAamvaBarcodeVerificationNative")
public class ScanditIdAamvaBarcodeVerificationNativePlugin extends Plugin {
    @PluginMethod
    public void echo(PluginCall call) {
        call.resolve();
    }
}
